### PR TITLE
rebrand docs for scylla

### PR DESCRIPTION
**Pull request by lauranovich**
Rebrands the ReadMe page for Scylla 
Changing Links to Scylla sources
Removes Datastax and Datastax enterprise

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,25 @@
 ```eval_rst
 :orphan:
 ```
-# Datastax Java Driver for Apache Cassandra®
+# Scylla Java Driver for Apache Cassandra®
```
Comment (haaawk) [inline README.md]: I would say "for Scylla and Apache Cassandra@"

File: README.md [hunk 2]
```diff
@@ -98,54 +91,38 @@ The 'extras' module is also published as a separate artifact:
...
 If you are upgrading from a previous version of the driver, be sure to have a look at
 the [upgrade guide](/upgrade_guide/).
 
-If you are upgrading to _DataStax Enterprise_, use the [DataStax Enterprise Java driver][dse-driver] for more
-features and better compatibility.
+
 
 ## License
-&copy; DataStax, Inc.
+&copy; ScyllaDB, all rights reserved.
```
Comment (haaawk) [inline README.md]: Is it legal to remove copyright like that? After all we're basing our version on their work. I'm not an expert but that sounds dangerous.

Comment (lauranovich) [reply README.md]: Probably so, perhaps we should comment it - so it is there but not visible. I don't like the fact we're copying to begin with

Comment (haaawk) [reply README.md]: If you want we can replace it totally. Feel free to prepare our own version of the README.

Comment (lauranovich) [reply README.md]: If we do this option, do I overwrite the existing readme, or have a new file called index-readme or similar and just point to it?what do you suggest?

File: README.md (hunk 1 above)
Comment (lauranovich) [inline README.md]:
```suggestion
# Java Driver for Scylla and Apache Cassandra®
```

File: README.md (hunk 2 above)
Comment (lauranovich) [reply README.md]: @tzach @haaawk  please chime in here

Comment (haaawk) [reply README.md]: Just override IMO.

File: README.md [hunk 3]
```diff
@@ -98,54 +91,37 @@ The 'extras' module is also published as a separate artifact:
...
-features and better compatibility.
 
-## License
-&copy; DataStax, Inc.
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
+## License
+&copy; ScyllaDB, all rights reserved.
```
Comment (lauranovich) [inline README.md]:
```suggestion
&copy; ScyllaDB, all rights reserved. &copy; DataStax, Inc.
```

**Comment by lauranovich** [comment]
@haaawk  - I added the datastax copyright back. If this is acceptable please let me know.

**Comment by lauranovich** [comment]
closing  - was already done in another PR

**Pull request closed without merging**